website_docs/getting_started/browser.md

```diff
@@ -47,7 +54,7 @@ npm init -y
 npm install --save @opentelemetry/web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
```

Comment:
> I know this isn't a part of your PR, but wanted to point it out. I actually ran into this issue while following the example bc I was on npm 6. I think the simplest thing is to just install it as a dep and gloss over it not being needed on v7. 

For me having a documentation that makes it easy for everyone to get started is essential, so that's definitely within the scope of this PR! I think having it installed explicitly is fine, I commit your suggestion. thanks